.github/scripts/toolchain/build-gcc-stage1.sh

```diff
@@ -32,12 +32,12 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$GCC_BUILD_PATH/Makefile" ] ; then
         case "$PLATFORM" in
             *linux*)
                 TARGET_OPTIONS="$TARGET_OPTIONS \
-                    --enable-languages=c,lto \
+                    --enable-languages=c \
```

Comment:
I know the answer but I want to also double check: Can you confirm that LTO support in stage 1 compiler is trully needed to successfully build LTO support for stage 2 compiler?

Reply:
stage 1 compiler is used to built mingw-crt which is without LTO now.
This minor refactoring removes lto from  --enable-languages which is misleading in the terms of LTO support.
--enable-lto should be used to support LTO.
To have mingw fully compiled with LTO it should be added to stage 1 compiler. 
Now mingw is compiled without LTO and it will be separate PR to support it.

Reply:
Then I don't understand why this PR is removing the LTO both from stage 1 and stage 2?

Reply:
stage 2 has support for LTO by using --enable-lto.
It looks like using --enable-lto is enough to enable it.

Reply:
As it was discussed offline, it looks like lto is not needed in --enable-languages, however it will stay there for aligning with Cygwin and MSYS2 configurations.